bindings/python/iota_sdk/types/burn.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations  # For Burn reference in Burn
+from dataclasses import dataclass
+from typing import List, Optional, Dict, Any
+from iota_sdk.types.native_token import NativeToken
+
+
+@dataclass
+class Burn:
+    """A DTO for [`Burn`]
+
+    Parameters:
+    -----------
+    aliases: Optional[List[str]]
+        The aliases to burn (hex encoded)
+    nfts: Optional[List[str]]
+        The NFTs to burn (hex encoded)
+    foundries: Optional[List[str]]
+        The foundries the burn (hex encoded)
+    nativeTokens: Optional[List[NativeToken]]
+        The native tokens of the burn
```

Comment:
"The native tokens to burn (type?)"

Reply:
thanks, fixed. I don't know how this happend :D. Type in this case is not necessary, becasuse, unlike `str`, `NativeToken` is specific